x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations_settings/curations_settings.tsx

```diff
@@ -110,11 +110,7 @@ export const CurationsSettings: React.FC = () => {
           </EuiButtonTo>
         }
       >
-        <EuiButtonEmpty
-          target="_blank"
-          iconType="popout"
-          href={`${docLinks.enterpriseSearchBase}/license-management.html`}
-        >
+        <EuiButtonEmpty target="_blank" iconType="popout" href={`${docLinks.licenseManagement}`}>
```

Comment:
```suggestion
        <EuiButtonEmpty target="_blank" iconType="popout" href={docLinks.licenseManagement}>
```
No need for a template literal here